src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,27 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  if (userInput.length !== 4) {
+    return false;
+  }
+
+  if (isNaN(userInput)) {
```

Comment:
The `isNaN` function can give incorrect results when used with strings that represent numbers. For example, `isNaN('1234')` will return false, which is correct, but `isNaN('0123')` will also return false, which may not be the intended behavior if leading zeros are not allowed. Consider using a regular expression to ensure that the input is a four-digit number with no leading zeros and each digit is unique.